server/auth_server.go

```diff
@@ -174,20 +174,18 @@ func readSettings(settings []string) (map[string]string, error) {
 
 func updateSettings(settings map[string]string) error {
 	for key, value := range settings {
-		if value != "" {
-			log.Debugf("Update setting key:%v value: %v", key, value)
-			setting, err := RancherClient.Setting.ById(key)
-			if err != nil {
-				log.Errorf("Error getting the setting %v , error: %v", key, err)
-				return err
-			}
-			setting, err = RancherClient.Setting.Update(setting, &client.Setting{
-				Value: value,
-			})
-			if err != nil {
-				log.Errorf("Error updating the setting %v to value %v, error: %v", key, value, err)
-				return err
-			}
+		log.Debugf("Update setting key:%v value: %v", key, value)
+		setting, err := RancherClient.Setting.ById(key)
+		if err != nil {
+			log.Errorf("Error getting the setting %v , error: %v", key, err)
+			return err
+		}
+		setting, err = RancherClient.Setting.Update(setting, &client.Setting{
+			Value: value,
+		})
+		if err != nil {
+			log.Errorf("Error updating the setting %v to value %v, error: %v", key, value, err)
+			return err
```

Comment:
Can you mention that this commit has dependency on go-rancher PR needed and there will be vendor folder changes needed accordingly.